core/src/main/resources/hudson/tasks/ArtifactArchiver/help-artifacts.html

```diff
@@ -5,4 +5,11 @@
    -- except that <tt>","</tt> (<em>comma</em>) is the only supported separator.
    The base directory is <a href='ws/'>the workspace</a>.
    You can only archive files that are located in your workspace.
+
+   An example on how to archive multiple artifacts from a specific folder:
+   <pre>archiveArtifacts: artifacts: 'target/*.jar'</pre>
```

Comment:
Note that this syntax is Pipeline-specific, and this help is shown in other contexts (e.g. freestyle). I would limit it to the specific string value specifying what to archive.

(Also, I think this is syntactically invalid, the first colon seems wrong?)